<launch>
   <!-- <node name="attack_node" pkg="attack" type="attack_task.py" output="screen"/> -->
   <!-- <node name="move_obj_node" pkg="attack" type="obs_obj.py" output="screen"/> -->
   <arg name="drone_id" default="1" />
   <arg name="drone_num" default="3" />
   <!-- <node name="attack_node" pkg="attack" type="attack_task.py" output="screen"/> -->
   <node pkg="attack" type="attack_task.py" name="attack_node_$(arg drone_id)" output="screen">
        <param name="drone_id" value="$(arg drone_id)"/>
        <param name="drone_num" value="$(arg drone_num)"/>
   </node>
   <!-- <node pkg="attack" type="obs_obj.py" name="move_obj_node_$(arg drone_id)" output="screen">
        <param name="drone_id" value="$(arg drone_id)"/>
        <param name="drone_num" value="$(arg drone_num)"/>
   </node> -->
   <!-- <node pkg="rflysim_ros_pkg" type="Rflysim_objwithsize.py" name="obj_control" output="screen">
        <param name="ip" value="$(arg ip)"/>
        <param name="mav_x" value="$(arg mav_x)"/>
        <param name="mav_y" value="$(arg mav_y)" />
        <param name="mav_z" value="$(arg mav_z)" />
   </node> -->
</launch>